src/nbl/video/CVulkanLogicalDevice.h

```diff
@@ -826,6 +767,27 @@ class CVulkanLogicalDevice final : public ILogicalDevice
         assert(retval == VK_SUCCESS);
     }
 
+    void* getExternalMemoryHandle(IDeviceMemoryBacked* obj) const override
+    {
+        if (!obj)
+            return nullptr;
+        auto mem = obj->getBoundMemory();
+        if (!mem)
+            return nullptr;
+        auto vkHandle = *static_cast<const VkDeviceMemory*>(mem->getNativeHandle());
+        if (!vkHandle)
+            return nullptr;
+        VkMemoryGetWin32HandleInfoKHR getHandleInfo = {
+            .sType = VK_STRUCTURE_TYPE_MEMORY_GET_WIN32_HANDLE_INFO_KHR,
+            .memory = vkHandle,
+            .handleType = VkExternalMemoryHandleTypeFlagBits(obj->getCachedCreationParams().externalHandleType.value),
+        };
+        void* handle = 0;
+        if(VK_SUCCESS == m_devf.vk.vkGetMemoryWin32HandleKHR(m_vkdev, &getHandleInfo, &handle))
```

Comment:
please guard Windows specific code with the appropriate `#ifdef NBL_....._PLATFORM_....`